wandb/apis/public.py

```diff
@@ -562,7 +562,7 @@ def _load_page(self):
 
     def __getitem__(self, index):
         loaded = True
-        while loaded and index > len(self.objects) - 1:
+        while loaded:
```

Comment:
This isnt the right fix... I think this will defeat the entire pagination system and will load everything rather than loading the page up to the index needed.

Reply:
I'm not clear on what the check is doing here.

Reply:
Looks like pagination always started at 0. And we grow the list infinitely?

Reply:
index is the element of the list we are looking for.  and we keep reading until we get a list of items that includes that index.

at least that is how i read it.